### PR TITLE
[EASY]: Hide 'shares purchased' on payment success screen when paying a bill

### PR DESCRIPTION
**Pull request by niwong**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/styles/PaymentSuccessCard.css [hunk 1]
```diff
@@ -54,3 +54,7 @@
   padding-bottom: 38px;
   margin: 21px 30px 30px 38px;
 }
+
+.payment-success-hide {
+  display: none !important;
```
Comment (ashleynguyen27) [inline src/styles/PaymentSuccessCard.css:59]: does this work without using !important?

Comment (niwong) [reply src/styles/PaymentSuccessCard.css:59]: It did not.

Comment (niwong) [reply src/styles/PaymentSuccessCard.css:59]: Let me try to make it work without important 😅

Comment (ashleynguyen27) [reply src/styles/PaymentSuccessCard.css:59]: ok just wondering. approved!

**Pull request merged**